src/sql/workbench/contrib/query/browser/queryEditor.ts

```diff
@@ -96,6 +96,7 @@ export class QueryEditor extends EditorPane {
 	private _actualQueryPlanAction: actions.ActualQueryPlanAction;
 	private _listDatabasesActionItem: actions.ListDatabasesActionItem;
 	private _toggleSqlcmdMode: actions.ToggleSqlCmdModeAction;
+	private _toggleActualExecutionPlanMode: actions.ToggleActualExecutionPlanModeAction;
```

Comment:
I don't know how useful the telemetry event for the toggle button is if we are sending an event every time user generates actual execution plan.

Reply:
Up to you, but generally it's good to know how many times people are using a UI feature (so we can determine if we can remove it later on if there aren't many people using it)